From e5e3232f7268110f7e3e3c4814eab31a6704b672 Mon Sep 17 00:00:00 2001
From: jiangpengfei <jiangpengfei9@huawei.com>
Date: Mon, 10 Aug 2020 20:11:07 +0800
Subject: [PATCH 22/50] kata-runtime: skip go version check and do not build
 containerd-shim-v2

reason: skip go version check and do not build containerd-shim-v2
because iSulad current not support shimV2

Signed-off-by: jiangpengfei <jiangpengfei9@huawei.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 14a0ea47..d5e4bbe1 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@ for file in /etc/os-release /usr/lib/os-release; do \
     fi \
 done)
 
-SKIP_GO_VERSION_CHECK=
+SKIP_GO_VERSION_CHECK=y
 include golang.mk
 
 #Get ARCH.
@@ -503,7 +503,7 @@ define SHOW_ARCH
   $(shell printf "\\t%s%s\\\n" "$(1)" $(if $(filter $(ARCH),$(1))," (default)",""))
 endef
 
-all: runtime containerd-shim-v2 netmon
+all: runtime netmon
 
 # Targets that depend on .git-commit can use $(shell cat .git-commit) to get a
 # git revision string.  They will only be rebuilt if the revision string
-- 
2.14.3 (Apple Git-98)

